tests/test_gunw.py

```diff
@@ -16,108 +18,149 @@ def test_granule_ur_pattern():
+                'checksum': 'e5094bda56e2316f2ec71d708cf1b4e6',
+                'checksumType': 'md5',
+            },
+        ],
+        'dataVersion': '1.0',
+    }
     expected = {
-        'ProductName': 'myFilename',
-        'Browse': {
-            'Bucket': 'myBucket',
-            'Key': 'myPrefix/myFilename.png',
-        },
-        'Metadata': {
-            'Bucket': 'myBucket',
-            'Key': 'myPrefix/myFilename.json',
-        },
-        'Product': {
-            'Bucket': 'myBucket',
-            'Key': 'myPrefix/myFilename.nc',
-        },
+        'identifier': 'myFilename',
+        'collection': 'ARIA_S1_GUNW',
+        'version': '1.6.1',
+        'submissionTime': '2025-02-18T01:02:03.000456Z',
+        'product': product,
+        'provider': 'ASF_HyP3',
+        'trace': 'ASF-TOOLS',
     }
-
+    print(gunw._generate_ingest_message(job))
     assert gunw._generate_ingest_message(job) == expected
 
 
-def test_publish_message():
-    with patch('boto3.client') as mock_client:
-        mock_sns = MagicMock()
-        mock_client.return_value = mock_sns
-
-        gunw._publish_message({'ProductName': 'foo'}, 'arn:aws:sns:us-east-1:123456789012:myTopic')
-
-        mock_client.assert_called_once_with('sns', region_name='us-east-1')
-        mock_sns.publish.assert_called_once_with(
-            TopicArn='arn:aws:sns:us-east-1:123456789012:myTopic',
-            Message='{"ProductName": "foo"}',
-        )
-
-    with patch('boto3.client') as mock_client:
-        mock_sns = MagicMock()
-        mock_client.return_value = mock_sns
-
-        gunw._publish_message({'ProductName': 'bar'}, 'arn:aws:sns:us-west-2:123456789012:myTopic')
+def test_process_job_if_not_archived(monkeypatch, s3_bucket, gunw_data_path):
+    monkeypatch.setenv('CMR_DOMAIN', 'cmr.earthdata.nasa.gov')
+    monkeypatch.setenv('GUNW_QUEUE_URL', 'myQueueUrl')
 
-        mock_client.assert_called_once_with('sns', region_name='us-west-2')
-        mock_sns.publish.assert_called_once_with(
-            TopicArn='arn:aws:sns:us-west-2:123456789012:myTopic',
-            Message='{"ProductName": "bar"}',
-        )
+    aws.S3_CLIENT.upload_file(str(gunw_data_path / 'metadata.json'), s3_bucket, 'myPrefix/myFilename.json')
+    aws.S3_CLIENT.upload_file(str(gunw_data_path / 'browse.png'), s3_bucket, 'myPrefix/myFilename.png')
+    aws.S3_CLIENT.upload_file(str(gunw_data_path / 'data.nc'), s3_bucket, 'myPrefix/myFilename.nc')
 
-
-def test_process_job_if_not_archived(monkeypatch):
-    monkeypatch.setenv('CMR_DOMAIN', 'cmr.earthdata.nasa.gov')
-    monkeypatch.setenv('INGEST_TOPIC_ARN', 'myTopicArn')
+    now = datetime.datetime(2025, 2, 18, 1, 2, 3, 456, tzinfo=datetime.UTC)
+    mock_datetime = MagicMock(wraps=datetime.datetime)
+    mock_datetime.now.return_value = now
+    monkeypatch.setattr(datetime, 'datetime', mock_datetime)
 
     job = {
+        'job_id': 'myPrefix',
         'job_type': 'ARIA_S1_GUNW',
-        'user_id': 'test-user',
```

Comment:
It still passes without `user_id`? Shouldn't this be required for the `_qualifies_for_ingest` function?

Reply:
Nevermind, it's because we refactored how the GUNW ingest configs work, so `ARIA_S1_GUNW` just has a `None` config.